java/pom.xml

```diff
@@ -85,6 +85,7 @@
     <surefire.version>3.0.0-M5</surefire.version>
     <test.tmp.dir>${project.build.directory}/testing-tmp</test.tmp.dir>
     <zstd-jni.version>1.5.6-4</zstd-jni.version>
+    <jts.version>1.19.0</jts.version>
```

Comment:
This seems to be too old because it's released on Jun 21, 2022.

Why don't we use the latest one? Is there any incompatibility issue with something?
- https://mvnrepository.com/artifact/org.locationtech.jts/jts-core/1.20.0